app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
```

Comment:
Built-in modules imports should be kept at the top of the file.
```suggestion
import datetime
from app.errors import NotVaccinatedError, \
                        OutdatedVaccineError, \
                        NotWearingMaskError
```